### DISCUSSION TITLE
Code Popup

### DISCUSSION THREAD

**Discussion by geomasoft** [Megabit/Blazorise#4803] [Ideas]
Hi Team!

A newbie here!

Blazorise is so clean and intuitive, thanks!

I was going through the demos on the Blazorise site and noticed there is a link to the GitHub repo for the whole demo.

I would prefer a snippet popup for each example so that it can be quickly studied.

Not sure if I am missing something or if another way of achieving the same thing already exists.

If it doesn't exist then I am more than happy to add it to the demo if I am shown how to.

**Comment by stsrki** [comment]
Hello @geomasoft 

The demo and documentation both have different purpose.

Documentation is meant to show code snippets with descriptions of what it does. It only contains very specific usage of the component.

With the demo apps, we want to show Blazorise in a somewhat "real-world" scenario. We also use it for our internal testing to see if the UI is properly rendered with our components. That's why we don't have any code snippets, as they would get in the way.

**Comment by geomasoft** [comment]
Excellent!!

Specific and focused examples are the best. I was just being lazy I guess,
just wanted go through the demo and if something impressed me or made me
curious then easily find how it is achieved by viewing the source code then
and there.

Blazorise is a genuine RAD tool and rapid learning would also be cool :)


Regards,

Reuben Alfred

------------------------------
*The information included in this email is reserved to named addressee’s
eyes only. You may not share this message or any of its attachments to
anyone. Please note that as the recipient, it is your responsibility to
check the email for malicious software. Finally, the opinions disclosed by
the sender do not have to reflect those of the company, therefore the
company refuses to take any liability for the damage caused by the content
of this email.*
